aliases.sh

```diff
@@ -149,7 +149,18 @@ function doubleline () {
 alias sizes="sudo du -cxhd 1"
 alias flushDNS='dscacheutil -flushcache'
 alias DSFiles_removal="find . -type f -name '*.DS_Store' -ls -delete"
-alias hosts_edit='sudo vi /etc/hosts'
+
+
+function hostedit() {
+    local apt=`command -v vim`
```

Comment:
Check the comment I left in the other commit.
Also, shouldn't `apt=` be `vim=` ?

Again, I still believe we could use the user's default editor, with a fallback to VI